File: Python/SivitasAkademik.py
from Human import Human


class SivitasAkademik(Human):
    __asal_universitas = ""
    __email_edu = ""

    def __init__(self, nik="", nama="", jenis_kelamin="", asal_universitas="", email_edu="",):
        super().__init__(nik, nama, jenis_kelamin)
        self.__asal_universitas = asal_universitas
        self.__email_edu = email_edu

    def set_asal_universitas(self, asal_universitas):
        self.__asal_universitas = asal_universitas

    def set_email_edu(self, email_edu):
        self.__email_edu = email_edu

    def get_asal_universitas(self):
        return self.__asal_universitas

    def get_email_edu(self):
        return self.__email_edu

File: CPP/SivitasAkademik.cpp
class SivitasAkademik : public Human
{
private:
    string asal_universitas;
    string email_edu;

public:
    SivitasAkademik(string nik = "", string nama = "", string jenis_kelamin = "", string asal_universitas = "", string email_edu = "") : Human(nik, nama, jenis_kelamin)
    {
        this->asal_universitas = asal_universitas;
        this->email_edu = email_edu;
    }
    void set_asal_universitas(string asal_universitas)
    {
        this->asal_universitas = asal_universitas;
    }
    void set_email_edu(string set_email_edu)
    {
        this->email_edu = email_edu;
    }
    string get_asal_universitas()
    {
        return this->asal_universitas;
    }
    string get_email_edu()
    {
        return this->email_edu;
    }
};
